### PR TITLE
Port existing cwitc community page into new cwitc website

### PR DESCRIPTION
**Pull request by daniel-packard**
Hey Josh - I took a swing at it

I'll need some guidance on how you want to tie it into the nav (hidden?), and the config.yml + cms bits.

**existing page:** https://www.cwitc.org/our-community
image

**new page:** (dev/our-community)
image

### CODE REVIEW

**Comment by jevenson** [comment]
@daniel-packard cool, thanks for working on this so fast. I'll try to review it today sometime.

Just FYI, you should be able to create and push branches in this repository, instead of forking. That will help out with feature branch deploys.

So if you create a branch in this repository and push it up, called something like `community`, then your changes would be visible online at `community--cwitc.netlify.com`

**Comment by daniel-packard** [comment]
Preview deploy was automatically created when I opened the PR (#11)
- https://deploy-preview-11--cwitc.netlify.com/

I prefer branching on my own fork when I'm working on a shared repo, but if it's the project guidelines I _guess_ i can change 😉

**Comment by daniel-packard** [comment]
PS - merged in the changes from master, but i'll have to review after work to make sure I didn't flub it

**Comment by daniel-packard** [comment]
Alright.. merge flubbed... and now repaired. 

Ready for review

File: src/site/styles/_our-community.scss [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    .icon {
+      width: 75%;
+      height: 100%;
+      fill: $brand-dark;
+    }
+  }
+
+  .page-hero {
+    background: url(/assets/about_hero_background.png);
```
Comment (daniel-packard) [inline src/site/styles/_our-community.scss]: I am hopeless with assets. I just used the same hero image as About, but we can swap in whatever

File: src/site/styles/_our-community.scss [hunk 2]
```diff
@@ -0,0 +1,34 @@
+@import "vars";
+
+.our-community {
```
Comment (daniel-packard) [inline src/site/styles/_our-community.scss]: A lot of these styles are duplicated from the home and about styles... not sure if we want to move them to a better shared location

File: src/site/our-community.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
+layout: our-community.hbs
+title: Central Wisconsin Community
+navText: Community
+hideFromNav: false
+changefreq: monthly
+priority: 1.0
```
Comment (jevenson) [inline src/site/our-community.md]: Set priority to 0.5

File: src/site/styles/_our-community.scss (hunk 1 above)
Comment (jevenson) [reply src/site/styles/_our-community.scss]: Yeah, we'll want to swap in the same image that's on the current community page

File: src/site/styles/_our-community.scss (hunk 2 above)
Comment (jevenson) [reply src/site/styles/_our-community.scss]: Sure, we can revisit that after this is merged in

**Comment by jevenson** [comment]
One small thing here, I'm not a fan of how the bullets are indented. This is a user agent style of having 40px of padding on the ul. If we override it to 20px, the bullets line up with the heading above. There is a reset css file where that style can be added.


image

File: src/site/styles/_our-community.scss (hunk 2 above)
Comment (daniel-packard) [reply src/site/styles/_our-community.scss]: created #12

File: src/site/styles/_our-community.scss [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  }
+
+  .page-hero {
+    background: url(/assets/our_community_hero_background.png), rgba(120,120,120,1);
+    background-blend-mode: multiply;
+    background-position: center bottom;
+    background-size: 100% auto;
+    height: 300px;
+    margin-bottom: 3rem;
+    color: #fff;
```
Comment (jevenson) [inline src/site/styles/_our-community.scss]: Please use `$brand-light` here

File: src/site/styles/_our-community.scss [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+    .icon {
+      width: 75%;
+      height: 100%;
+      fill: $brand-dark;
+    }
+  }
+
+  .page-hero {
+    background: url(/assets/our_community_hero_background.png), rgba(120,120,120,1);
```
Comment (jevenson) [inline src/site/styles/_our-community.scss]: Please use the lighten function on the `$brand-dark`.

Comment (daniel-packard) [reply src/site/styles/_our-community.scss]: ahhh, good call 👍

File: src/site/styles/_our-community.scss [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+  }
+
+  .page-hero {
+    background: url(/assets/our_community_hero_background.png), lighten($brand-dark, 40);
+    background-blend-mode: multiply;
+    background-position: center bottom;
+    background-size: 100% auto;
+    @include media-breakpoint-down(sm) {
+      background-size: auto 100%;
+    }
```
Comment (daniel-packard) [inline src/site/styles/_our-community.scss]: I pulled one more thread here.. added a breakpoint so the hero image doesn't tile vertically on small screens.

Comment (jevenson) [reply src/site/styles/_our-community.scss]: Still getting a bit of tiling. try setting the background size to be `background-size: cover;` for all screen sizes. The about page has this issue too, so if we can potentially share a style here, that would be good.


image

Comment (daniel-packard) [reply src/site/styles/_our-community.scss]: fixed

This is witchcraft -- glad I know someone who has listed CSS as a skill on their linked in profile!

**Comment by jevenson** [comment]
Thanks Dan!

**Pull request merged**